source/_app/helpers/utils.js

```diff
@@ -70,6 +70,13 @@ app.utils = {
   isNumeric: function(string) {
     var hasNumber = /^\d+$/;
     return hasNumber.test(string);
-  }
+  },
+
+  /**
+   * Pad out numbers < 10
+   */
+  pad: function(n) {
+    return (n < 10) ? ('0' + n) : n;
```

Comment:
ah, I decided not to use this, I've made a note to delete this method when I merge the PR, the idea was that when  you enter a number in the hour/minute fields  < 10 it padded it out so it looked neater 01 instead of 1, but it didn't work particularly well